src/status_im/ui/screens/profile/contact/views.cljs

```diff
@@ -1,48 +1,54 @@
 (ns status-im.ui.screens.profile.contact.views
   (:require [re-frame.core :as re-frame]
+            [reagent.core :as reagent]
             [status-im.i18n :as i18n]
             [status-im.ui.components.list.views :as list]
+            [status-im.utils.utils :as utils]
             [status-im.ui.components.react :as react]
             [status-im.ui.components.status-bar.view :as status-bar]
+            [status-im.ui.components.large-toolbar.view :as large-toolbar]
             [status-im.ui.components.toolbar.view :as toolbar]
-            [status-im.ui.screens.profile.components.styles :as profile.components.styles]
             [status-im.ui.screens.profile.components.views :as profile.components]
             [status-im.ui.screens.profile.contact.styles :as styles]
-            [status-im.utils.platform :as platform])
+            [status-im.ui.components.list-item.views :as list-item]
+            [status-im.ui.components.list-header.views :as list-header]
+            [status-im.ui.components.chat-icon.screen :as chat-icon]
+            [status-im.ui.components.icons.vector-icons :as vector-icons]
+            [status-im.ui.screens.profile.components.sheets :as sheets]
+            [status-im.utils.identicon :as identicon]
+            [status-im.ui.screens.chat.photos :as photos]
+            [status-im.multiaccounts.core :as multiaccounts])
```

Comment:
not sure if you use emacs call cljr-refactor clean-ns on that namespace to remove unused deps and order them alphabetically, there should be a similar feature in cursive

Reply:
I actually use vscode with https://marketplace.visualstudio.com/items?itemName=betterthantomorrow.calva
need to find something similar..

Reply:
@yenda cant find anything.. do you know some command line plugin ? i use `lein cljfmt fix <>` to do final formatting but doesn't fix namespaces...

Reply:
i removed them manually...